mne_neuron/dipole.py

```diff
@@ -245,3 +244,19 @@ def baseline_renormalize(self, params):
         # recalculate the aggregate dipole based on the baseline
         # normalized ones
         self.dpl['agg'] = self.dpl['L2'] + self.dpl['L5']
+
+    def write(self, fname = 'dpl.txt'):
+        """Write dipole values to a file.
+
+        Parameters
+        ----------
+        fname : str
+            Output filename. Must be fully specified
+        """
+
+        with open(fname, 'w') as f:
+            for t, x_agg, x_L2, x_L5 in zip(self.t, self.dpl['agg'], self.dpl['L2'], self.dpl['L5']):
+                f.write("%03.3f\t" % t)
+                f.write("%5.4f\t" % x_agg)
+                f.write("%5.4f\t" % x_L2)
+                f.write("%5.4f\n" % x_L5)
```

Comment:
could we use [np.savetxt](https://docs.scipy.org/doc/numpy/reference/generated/numpy.savetxt.html) here to do this more compactly?

Reply:
Need this for a diff against the old version of HNN.

Reply:
sure, but couldn't you do that with `np.savetxt` ? Or do you want to first go with this for the sake of reproducibility and then slowly replace things with more compact versions while ensuring reproducibility? That's fine by me, but could you raise an issue so that we can keep track of things that can be improved once we are sure that MNE-neuron is giving you exactly the same result as HNN.